lib/rules/require-padding-newlines-before-line-comments.js

```diff
@@ -52,8 +52,20 @@ module.exports.prototype = {
                 return;
             }
 
+            var prevToken = file.getPrevToken(comment, {
+                includeComments: true
+            });
+
+            if (prevToken.type === 'Line') {
+                return;
+            }
+
+            if (prevToken.type === 'Punctuator' && prevToken.value === '{') {
```

Comment:
@hshoff: Can you weigh in here? Would Airbnb want an override that wouldn't require a blank line before a comment if it is the first in the block? 

Aka, should this be an option at all or just always require a blank line?